### PR TITLE
Updates UG command format and demo screenshots.

### LINKED ISSUE #217: [PE-D] UG Image for deleteRole wrong capitalisation
Minor issue with the user guide on the deleteRole Use Case #2.1 on the capitalisation which I encountered till I look further into the documentation. 

From documentation:

Actual:

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: whoisjunhong/ped#1

### LINKED ISSUE #255: Fix UG add company
image

### PR DESCRIPTION
**Pull request by justinyjt**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -448,6 +448,7 @@ companies less clustered and thus more organised.
 * Refer to [Demo Use Case #2](#usecase2) for an example use case.
 
 **Examples:**
+>>>>>>> 183b65a12c384a90ad9bcfc0ea458d5e32a94af1
```
Comment (BrendonLau) [inline docs/UserGuide.md]: I think you forget to remove this line after conflict.

**Review by BrendonLau — APPROVED** [review]
Other than the minor thing, LGTM

**Review by kerbysoh — APPROVED** [review]
LGTM!

File: docs/UserGuide.md (hunk 1 above)
Comment (justinyjt) [reply docs/UserGuide.md]: Good catch thanks!

**Pull request merged**